src/lib.rs

```diff
@@ -203,23 +203,58 @@ pub struct Metrics {
     pub strikeout_thickness: f32,
 }
 
-pub trait Rasterize {
-    /// Errors occurring in Rasterize methods.
-    type Err: ::std::error::Error + Send + Sync + 'static;
+/// Errors occuring when using the rasterizer.
+#[derive(Debug)]
+pub enum Error {
+    /// Couldn't find font matching description.
+    MissingFont(FontDesc),
+
+    /// Tried to get size metrics from a Face that didn't have a size.
+    MissingSizeMetrics,
+
+    MissingGlyph(RasterizedGlyph),
+
+    /// Requested an operation with a FontKey that isn't known to the rasterizer.
+    FontNotLoaded,
```

Comment:
Instead of saying that the font isn't loaded, shouldn't we say that the glyph key isn't valid? Since that's how the enduser interacts with our API.

Since it's `get_glyph(GLYPH_KEY)`, I wouldn't expect the error to tell me that the font isn't loaded, but just that the glyph key is invalid. For whatever reason that is, is up to me?